corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -240,8 +246,12 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
                 collection: listCollection,
                 headers: detailObject.get('headers'),
                 styles: detailObject.get('styles'),
+                tiles: detailObject.get('tiles'),
                 title: detailObject.get('title'),
             };
+            if (detailObject.get('usesCaseTiles')) {
+                return views.CaseTileDetailView(menuData);
```

Comment:
I think `CaseDetailTileView` would be more descriptive if we could also rename `CaseListDetailView` to `CaseDetailTableView`

Reply:
Tweaked colors ever so slightly to be more standard: 39cf74a9778